src/FMBot.Bot/Services/Guild/GuildService.cs

```diff
@@ -164,6 +164,37 @@ public async Task<List<UserExportModel>> FindAllUsersFromGuildAsync(ICommandCont
             return usersObject.ToList();
         }
 
+        public async Task StaleGuildLastIndexedAsync(IGuild guild)
+        {
+            await using var db = this._contextFactory.CreateDbContext();
+            var existingGuild = await db.Guilds
+                .AsQueryable()
+                .FirstOrDefaultAsync(f => f.DiscordGuildId == guild.Id);
+
+            if (existingGuild == null)
+            {
+                var newGuild = new Persistence.Domain.Models.Guild
+                {
+                    DiscordGuildId = guild.Id,
+                    Name = guild.Name,
+                    TitlesEnabled = true,
+                };
+
+                await db.Guilds.AddAsync(newGuild);
```

Comment:
You can't come here if the guild doesn't exist in the database, so don't think this is needed